__tests__/index.test.js

```diff
@@ -180,3 +180,41 @@ describe("GET /api/articles/:article_id/comments", () => {
   });
 
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("status 201: responds with the posted comment object", () => {
+      return request(app)
+          .post("/api/articles/1/comments")
+          .send({username: "butter_bridge", body: "This is a test comment"})
+          .expect(201)
+          .then(({body}) => {
+              expect(body.comment_id).toBe(19);
+              expect(body.author).toBe("butter_bridge");
+              expect(body.article_id).toBe(1);
+              expect(body.body).toBe("This is a test comment");
+          });
+  });
+
+  //Error handling
+  test("status 404: responds with error message when the given article_id does not exist", () => {
+      return request(app)
+          .post("/api/articles/1000/comments")
+          .send({username: "butter_bridge", body: "This is a test comment"})
+          .expect(404)
+          .then(({body}) => {
+              expect(body.msg).toBe("Article not found");
+          });
+  });
+
+  test("status 400: responds with error message when given an invalid article_id", () => {
+      return request(app)
+          .post("/api/articles/invalid/comments")
+          .send({username: "butter_bridge", body: "This is a test comment"})
+          .expect(400)
+          .then(({body}) => {
+              expect(body.msg).toBe("Bad request");
+          });
+  });
```

Comment:
Please add another test for a bad request where the object is missing required field(s), e.g. no username or body properties.